src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -5,7 +5,6 @@
 using System.Reflection;
 using System.Collections;
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2112:SecuredTypesShouldNotExposeFields", Scope = "type", Target = "System.ComponentModel.AttributeCollection")]
```

Comment:
But it's triggered by the presence of a link demand, right? Which we don't have. It doesn't just trigger on all `public or protected type contains public fields`